aip/general/0156.md

```diff
@@ -57,12 +59,15 @@ rpc UpdateConfig(UpdateConfigRequest) returns (Config) {
 - Singleton resources **should** define the [`Get`][aip-131] and
   [`Update`][aip-134] methods, and **may** define custom methods as
   appropriate.
+  - However, singleton resources **must not** define the [`Update`][aip-134]
```

Comment:
nit: probably should move the update guidance to it's own line (e.g. singleton resources should define update unless all fields are output-only).

Although It seems like we could just refactor this guidance to just pointing to the update method in general.